frontend/src/components/common/datepicker/styles.module.scss

```diff
@@ -37,6 +37,7 @@
 }
 
 .errorMessage {
+  position: absolute;
```

Comment:
Should it be absolute? We don't have this in input styles

Reply:
@Liza-Veis I had to add absolute so it doesn't move other elements on datepicker field click

Reply:
right, we don't want to use an absolute

Reply:
right, we don't want to use an absolute

Reply:
right, we don't want to use an absolute